### ISSUE TITLE
Uploads dont work with browsersync 

### ISSUE THREAD

**Issue by ducopro** [livewire/livewire#1130]
### Description

The signed url for the file upload is always prefixed with the APP_URL

However, when using browsersync, this wont work.
Easy solution would be to use relative url for the upload.

**Context**
- Livewire version: [e.g. 1.2.0]
- Laravel version: [e.g. 7.0.0]
- Browser: [e.g. Chrome]

**Comment by basepack** [comment]
Same here, I just tested them with the app open on the real URL. But it is not ideal.

**Comment by keenminded** [comment]
I can confirm the same behavior.

**Comment by basepack** [comment]
Below the error I got while using BrowserSync, as documented here: https://github.com/livewire/livewire/issues/1134

After a normal POST form request with embedded Livewire components as the (file upload) inputs this error was thrown:

```
Livewire\Exceptions\CorruptComponentPayloadException
Livewire encountered corrupt data when trying to hydrate the [input.embedded-image-upload] component. Ensure that the [name, id, data] of the Livewire component wasn't tampered with between requests.
```

Form was like: 

```
<form method="post">
   <input name="title"> <!-- normal input --!>

  <livewire:embedded-image-upload :currentImage="$currentImage"/> <!-- livewire component --!>

   <input type="submit" value="Save"> <!-- notice not saving via Livewire, but normal POST request --!>
</form>
```

**Comment by keenminded** [comment]
@calebporzio Hey there, Caleb. I noticed you closed both this and #1134 which are related. Are there any plans to have some type of workaround or fix for having a way to test file uploads when using BrowserSync or none at all?
Thank you in advance.

**Comment by squishythejellyfish** [comment]
👋 Oh Hi! I'm Squishy, the friendly jellyfish that manages Livewire issues.

I see this issue has been closed.

Here in the Livewire repo, we have an "issues can be closed guilt-free and without explanation" policy.

If for ANY reason you think this issue hasn't been resolved, PLEASE feel empowered to re-open it.

Re-opening actually helps us track which issues are a priority.

Reply "REOPEN" to this comment and we'll happily re-open it for you!

(More info on this philosophy here: https://twitter.com/calebporzio/status/1321864801295978497)
